src/main/java/uk/gov/ons/ctp/common/domain/EstabType.java

```diff
@@ -29,24 +27,19 @@
   APPROVED_PREMISES("APPROVED PREMISES", AddressType.CE, SecurityType.SECURE),
   ROUGH_SLEEPER("ROUGH SLEEPER", AddressType.CE, SecurityType.NOT_SECURE),
   STAFF_ACCOMMODATION("STAFF ACCOMMODATION", AddressType.CE, SecurityType.NOT_SECURE),
-  CAMPHILL("CAMPHILL", AddressType.CE, SecurityType.NOT_SECURE),
-  HOLIDAY_PARK("HOLIDAY PARK", AddressType.CE, SecurityType.NOT_SECURE),
+
```

Comment:
Why add the empty line? Maybe just remove it?

Reply:
done